plugins/ohifv3/extensions/monai-label/src/services/MonaiLabelClient.js

```diff
@@ -10,28 +23,18 @@ export default class MonaiLabelClient {
     return await MonaiLabelClient.api_get(url.toString());
   }
 
-  async segmentation(model, image, params = {}, label = null) {
-    // label is used to send label volumes, e.g. scribbles,
-    // that are to be used during segmentation
-    return this.infer(model, image, params, label);
-  }
-
-  async deepgrow(model, image, foreground, background, params = {}) {
-    params['foreground'] = foreground;
-    params['background'] = background;
-    return this.infer(model, image, params);
-  }
-
   async infer(model, image, params, label = null, result_extension = '.nrrd') {
+    console.log('Running Infer for: ', model, image, params);
+
     let url = new URL('infer/' + encodeURIComponent(model), this.server_url);
     url.searchParams.append('image', image);
-    url.searchParams.append('output', 'all');
-    // url.searchParams.append('output', 'image');
+    // url.searchParams.append('output', 'all');
+    url.searchParams.append('output', 'image');
```

Comment:
is there a reason for this change from 'all' -> 'image', consider adding a comment

Reply:
monailabel server can output image and other information (json) and currently we don't consume other information for inference.